<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment> All rights reserved. </comment>
 <class>CQPlotSubwidget</class>
 <widget class="CopasiWidget" name="CQPlotSubwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>127</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>Type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkLogX">
       <property name="text">
        <string>log X-axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkLogY">
       <property name="text">
        <string>log Y-axis</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelScale">
     <property name="text">
      <string>Axis scales</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="typeLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>2D Plot</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Plot &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="layoutTitle">
     <item>
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="activeCheckBox">
       <property name="text">
        <string>active?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="Line" name="line1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="layoutCurves">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="labelCurves">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Curve specifications</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerCurves">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addCurveButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New Curve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addHistoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New Histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteCurveButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="Line" name="lineButton">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QHBoxLayout" name="layoutButtons">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="startPlotButton">
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copyPlotButton">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPlotButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePlotButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QListWidget" name="mpListPlotItems">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
     <widget class="QStackedWidget" name="mpStack"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>addCurveButton</sender>
   <signal>clicked()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>addCurveSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addHistoButton</sender>
   <signal>clicked()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>addHistoSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteCurveButton</sender>
   <signal>clicked()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>removeCurve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startPlotButton</sender>
   <signal>clicked()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>commitPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>resetPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPlotButton</sender>
   <signal>clicked()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>addPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deletePlotButton</sender>
   <signal>clicked()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>deletePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpListPlotItems</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>itemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyPlotButton</sender>
   <signal>clicked()</signal>
   <receiver>CQPlotSubwidget</receiver>
   <slot>copyPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCurveSlot()</slot>
  <slot>addHistoSlot()</slot>
  <slot>removeCurve()</slot>
  <slot>commitPlot()</slot>
  <slot>deletePlot()</slot>
  <slot>addPlot()</slot>
  <slot>resetPlot()</slot>
  <slot>typeChanged()</slot>
  <slot>itemSelectionChanged()</slot>
  <slot>copyPlot()</slot>
 </slots>
</ui>
